### PR TITLE
Add commands for actions

### PR DESCRIPTION
**Pull request by apmasell**
These are things that can be issues to actions to get them do something action-specific

### CODE REVIEW

File: shesmu-server/src/main/resources/ca/on/oicr/gsi/shesmu/shesmu.js [hunk 1]
```diff
@@ -1912,6 +1912,57 @@ function nextPage(query, targetElement, onActionPage) {
...
+                      )
+                    );
+                    const image = document.createElement("IMG");
+                    dialog.appendChild(image);
+                    image.src = "indifferent.gif";
+                  } else if (count > 1) {
+                    const dialog = makePopup();
+                    dialog.appendChild(
+                      document.createTextNode(
+                        `Purged ${count} actions!!! This is awkward. The unique action IDs aren't unique!`
```
Comment (callunity) [inline shesmu-server/src/main/resources/ca/on/oicr/gsi/shesmu/shesmu.js]: Is this text accurate in the context of commands?

**Comment by callunity** [comment]
Deploying this shows a new-to-me error in the logs, not sure if it's connected to these changes:
```
com.fasterxml.jackson.core.JsonParseException: Unexpected character ('<' (code 60)): expected a valid value (number, String, array, object, 'true', 'false' or 'null')
 at [Source: (org.apache.http.conn.EofSensorInputStream); line: 1, column: 2]
	at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:1804)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:693)
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportUnexpectedChar(ParserMinimalBase.java:591)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._handleUnexpectedValue(UTF8StreamJsonParser.java:2630)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser._nextTokenNotInObject(UTF8StreamJsonParser.java:832)
	at com.fasterxml.jackson.core.json.UTF8StreamJsonParser.nextToken(UTF8StreamJsonParser.java:729)
	at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4142)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4001)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3071)
	at ca.on.oicr.gsi.shesmu.core.actions.fake.RemoteInstance.update(RemoteInstance.java:48)
	at ca.on.oicr.gsi.shesmu.core.actions.fake.RemoteInstance.update(RemoteInstance.java:20)
	at ca.on.oicr.gsi.shesmu.plugin.json.JsonPluginFile.update(JsonPluginFile.java:38)
	at ca.on.oicr.gsi.shesmu.server.plugins.PluginManager$FormatTypeWrapper$FileWrapper.update(PluginManager.java:736)
	at ca.on.oicr.gsi.shesmu.plugin.files.AutoUpdatingDirectory$1.update(AutoUpdatingDirectory.java:53)
	at ca.on.oicr.gsi.shesmu.plugin.files.FileWatcher$RealFileWatcher$RetryProcess.update(FileWatcher.java:44)
	at ca.on.oicr.gsi.shesmu.plugin.files.FileWatcher$RealFileWatcher.run(FileWatcher.java:166)
	at java.lang.Thread.run(Thread.java:748)
```

File: shesmu-server/src/main/resources/ca/on/oicr/gsi/shesmu/shesmu.js (hunk 1 above)
Comment (apmasell) [reply shesmu-server/src/main/resources/ca/on/oicr/gsi/shesmu/shesmu.js]: Half of it. ;-) Fixed.

**Comment by apmasell** [comment]
That error is probably due to having incorrect `sbp-seqware-production`... The `/actions` endpoint will require a password, so the fake actions won't work. I will un-protect that endpoint among others.

**Comment by callunity** [comment]
Sounds good!

**Pull request merged**